static/app/views/settings/organizationMembers/inviteBanner.spec.tsx

```diff
@@ -0,0 +1,173 @@
+import {browserHistory} from 'react-router';
+import moment from 'moment';
+
+import {act, render, screen} from 'sentry-test/reactTestingLibrary';
+
+import {MissingMember} from 'sentry/types';
+import {DEFAULT_SNOOZE_PROMPT_DAYS} from 'sentry/utils/promptIsDismissed';
+import {InviteBanner} from 'sentry/views/settings/organizationMembers/inviteBanner';
+
+const missingMembers = {
+  integration: 'github',
+  users: TestStubs.MissingMembers() as MissingMember[],
+};
+
+const noMissingMembers = {
+  integration: 'github',
+  users: [] as MissingMember[],
+};
+
+describe('inviteBanner', function () {
+  beforeEach(function () {
+    MockApiClient.clearMockResponses();
+    MockApiClient.addMockResponse({
+      url: '/organizations/org-slug/missing-members/',
+      method: 'GET',
+      body: [],
+    });
+    MockApiClient.addMockResponse({
+      url: '/prompts-activity/',
+      method: 'GET',
+      body: {
+        dismissed_ts: undefined,
+        snoozed_ts: undefined,
+      },
+    });
+    (browserHistory.push as jest.Mock).mockReset();
+  });
+
+  it('render banners with feature flag', async function () {
+    const org = TestStubs.Organization({
+      features: ['integrations-gh-invite'],
+    });
+
+    render(
+      <InviteBanner
+        missingMembers={missingMembers}
+        onSendInvite={() => undefined}
+        organization={org}
+      />
+    );
+
+    await act(tick);
```

Comment:
Why do you need this? Can you instead use a `await screen.findBy...` and it will wait for the element to render.

Reply:
i get this error
```
    Warning: An update to InviteBanner inside a test was not wrapped in act(...).

    When testing, code that causes React state updates should be wrapped into act(...):

    act(() => {
      /* fire events that update state */
    });
    /* assert on the output */
```

Reply:
even with` await screen.findBy...`